docs/source/api/index.rst

```diff
@@ -2,344 +2,138 @@
+    directory component may be longer. Attempting to create paths with
+    longer components results in a ``400 Bad Request`` error response.
+  - Ordinarily, object names are opaque strings, and the Swift paths
+    ``/v1/AUTH_test/foo/bar``, ``/v1/AUTH_test/foo/./bar``,
+    ``/v1/AUTH_test/foo//bar``, and ``/v1/AUTH_test/foo/baz/../bar`` are
+    all distinct. In a filesystem, ``.`` and ``..`` have special meanings,
+    which would lead these to all map to the same file. To prevent
+    unintentional overwrites, only the first name is valid in a ProxyFS
+    account. Attempting to create an object at the other names results in
+    a ``400 Bad Request`` error response.
+
+- Object ETags (MD5 digests) are written as sysmeta, similar to what was
+  done for Swift's encryption and SLO features. If the client provides an
+  ETag header in a PUT request, it will be verified before finalizing the
+  write.
+- Object metadata must be valid UTF-8. Ordinarily, Swift only requires
+  UTF-8 for account and container metadata.
+- Object creation does not support the ``If-None-Match:*`` header.
+- Objects do not support expiration (though the ``X-Delete-At`` and
+  ``X-Delete-After`` headers will be written down).
+
+
+Reading objects via the object API
+----------------------------------
+
+- Account-level container counts, object counts, and total bytes used
+  reflect the storage of the underlying ProxyFS log segments and
+  checkpoints. As such, the counts are not particularly useful and the
+  bytes-used is only a crude approximation.
+- When listing an account, ``delimiter``, ``prefix``, and ``reverse``
+  query params are ignored.
+- When querying containers, the ``X-Storage-Policy`` header returns the
+  default storage policy for the cluster, which is not necessarily the
+  storage policy being used for log segment containers or the checkpoint
+  container.
+- Container-level object counts and total bytes used are always 0.
+- When listing a container, the ``reverse`` query parameter is ignored.
+  Only ``/`` is supported for ``delimiter``.
+- Auto-created directory objects are presented as zero-byte objects
+  with ``Content-Type: application/directory``.
+
+
+Reading objects via the file API
+--------------------------------
+
+- Account, container, and object metadata are inaccessible via the file API.
+- Large objects are only assembled via the object API. DLOs or SLOs, when
+  accessed through the filesystem API, will be zero-byte files or JSON
+  (respectively), similar to what happens when including a
+  ``?multipart-manifest=get`` query parameter via the object API.
+- Swift "symlink" objects are only followed when accessed through the object
+  API. When accessed through the filesystem API, these will be zero-byte
+  files.
+
+Reading files via the object API
+--------------------------------
+
+- Since containers map to directories, files written at the root of the
+  volume are not accessible via the object API.
```

Comment:
This maybe deserves a note about snapshots -- if you take a snapshot, you can get to the root-level file via the snapshot.